openssh/Makefile

```diff
@@ -22,7 +22,7 @@
 # Copyright 2019 Joyent, Inc.
 #
 
-VER =	openssh-portable-8.6p1
+VER =	openssh-8.8p1
```

Comment:
Should this still be openssh-portable?

Reply:
The download from openssh.org doesn't say portable, but I can rename and fix this, sure.